.github/workflows/build.yml

```diff
@@ -1,16 +1,16 @@
 name: build
 
 on:
-   #push:
-      #branches: [master]
+   push:
+      branches: [master]
```

Comment:
these being commented out was actually intentional, as there currently isn't really a big point in having automatic builds on master. Given that there is no binary being autopublished, and that master is already in master, having it autobuild pretty much just wastes CI minutes.

on PRs, it serves as a good way to make sure the code compiles and is fine, thus there I'd want to have the auto compilation

Also, generally, this really should go into a separate PR, no?

Reply:
yes i know. i am stupid

Reply:
i was testing stuff with the CI and forgot that i had a open PR, and didn't make a new branch. Sorry